threatmodels/notation-threatmodel.md

```diff
@@ -9,18 +9,18 @@ The following diagram illustrates the architecture and components.
+- **Artifact Builder**: The actor who is responsible for producing software artifacts using build systems. Notation supports signing and verification of software artifacts including but are not limited to container images, helm charts, Software Bill of Materials (SBOMs). These artifacts can be stored either in remote registries or locally on disk using [OCI image layout](https://github.com/opencontainers/image-spec/blob/v1.0.0/image-layout.md).
 - **Execution Environment**: The execution environment includes the host and **File System** where notation CLI will be installed and executed. Notation works with a shared responsibility model which means users/organizations are responsible for securing the notation execution environment. Following are the various directories and files used by the Notation:
   - `config.json` file is used to store various configurations such as credential store information, etc.
-  - `trustpolicy.json` file is used to store trust policy-related data.
-  - `truststore` directory is used to store various trusted certificates used for verifying the signature.
+  - `trustpolicy.json` file is used to store trust policy related data.
+  - `truststore` directory is used to store various trusted certificates used for signature verification.
   - `localkeys` directory is used to store test keys.
   - `plugins` directory is used to store various plugins.
   
   Notation uses credential stores to securely store the registry credentials.
 - **Notation Plugin**: An external component/binary that can integrate as one of the steps in Notation’s workflow for signature generation or verification, see [plugin spec](https://github.com/notaryproject/notaryproject/blob/v1.0.0-rc.2/specs/plugin-extensibility.md) for details.
-- **Registry**: A registry stores container images as well as related content formats, such as Helm charts, OCI artifacts built to the OCI image format specification. Registries are outside Notation trust boundary.
-- **KMS**: A Key Management System that supports the generation and management of signing material such as certificates (along with private key) management or a signing service. KMS are outside Notation trust boundary.
+- **Registry**: An OCI-compliant registry that stores OCI artifacts, like container images, Helm charts or other OCI artifacts. Registries are outside Notation trust boundary.
+- **KMS**: A Key Management System that manages the the certificate along with private key that used for signing artifacts or a signing service. KMS are outside Notation trust boundary.
```

Comment:
```suggestion
- **KMS**: A Key Management System that manages the the certificate along with private key that used for signing artifacts and provides a signing service. KMS are outside Notation trust boundary.
```

Reply:
@shizhMSFT I prefer to update it based on Pritesh's comment https://github.com/notaryproject/notaryproject/pull/242#discussion_r1242933122